lib/tasks/feature_flags.rake

```diff
@@ -5,4 +5,13 @@ namespace :feature_flags do
   task seed: :environment do
     FeatureFlagFactory.call
   end
+
+  desc "Activates development feature toggles for testing environment."
+  task activate_dev_toggles: :environment do
```

Comment:
This is really useful! I've been meaning to do something similar for local development too.

I think we should make this task depend on the `seed` task above, add it to `seeds.rb` and name it something like `enable_for_development`.

I'm wondering if it's worth extracting this out to a separate PR?

Reply:
I've raised it as https://github.com/nhsuk/manage-vaccinations-in-schools/pull/5482 because I also wanted to include a check to make sure this would never run outside of `RAILS_ENV=development`.